### PR TITLE
Upgrade rack-mini-profiler to v2.3.3

### PR DESCRIPTION
**Pull request by rsb**
Upgrade rack-mini-profiler to v2.3.3 to fix the ActionView::Template:Error (undefined method `safe_append=' for {:add_to_stack=>true}:Hash). 

Followed the advice from [here](https://github.com/rails/rails/issues/42261#issuecomment-930027308)

### Risks
- Low.

### CODE REVIEW

**Review by grosser — APPROVED** [review]
ah nice

File: Gemfile [hunk 1]
```diff
@@ -101,7 +101,7 @@ group :debugging do
 end
 
 group :development, :staging do
-  gem 'rack-mini-profiler'
+  gem 'rack-mini-profiler', '~> 2.3.2'
```
[the remark this answers is not in the document]
Comment (grosser) [reply Gemfile:104]: we don't want to block future upgrades

**Comment by grosser** [comment]
ah nice I though I already had bumped that 🤦 
... maybe this + what just got merged to master will fix things

**Comment by grosser** [comment]
deployed to staging https://samson.zende.sk/projects/samson/deploys/2100282

File: Gemfile (hunk 1 above)
[the remark this answers is not in the document]
Comment (rsb) [reply Gemfile]: ok removed the guard and ran `bundle update rack-mini-profiler` which bumped to 3.3.0. I tested that is it seemed to work

**Comment by grosser** [comment]
streaming deploys seems to still be broken, but this is a good step forward

**Comment by rsb** [comment]
I will start troubleshooting that now

**Pull request merged**